airbyte-server/Dockerfile

```diff
@@ -7,9 +7,7 @@ ENV APPLICATION airbyte-server
 
 WORKDIR /app
 
-COPY build/distributions/${APPLICATION}-0*.tar ${APPLICATION}.tar
-
-RUN tar xf ${APPLICATION}.tar --strip-components=1
+ADD build/distributions/${APPLICATION}-0.30.23-alpha.tar /app
```

Comment:
just curious, would we get the same effect if we deleted the tar archive after running tar xf?

Reply:
Maybe that combined with multistage builds, but just doing that directly still keeps the separate layers which are still part of the image and downloaded.

Reply:
double checking this will also pull in dev changes.

my understanding is we always build the tar named at the current version, regardless of dev or not, so this doesn't affect our current problem where we run into gradle caching when trying to build `dev`, where older versions are pulled in as the build task is cached.

so this should be an okay change. is this right?

Reply:
Yes, that's right.

Reply:
If we do change how that's done we may need to inject a `VERSION` environment variable here, but since it works for both dev and normal versions now, I'm inclined to leave it as-is until we get to that point. It should obviously fail on CI if this becomes in a problem so I feel like it's pretty safe.

Reply:
Folks as you know we reverted yestarday solution with `ADD` because this command unpack not the same way as we do with `tar xf ${APPLICATION}.tar --strip-components=1 `

another option to reduce image size is to use **Multi-stage**  build but the down-side that `Dockerfile` a become a little more complicated:

```
FROM airbyte/integration-base-java:dev AS build

WORKDIR /airbyte
ENV APPLICATION destination-dynamodb

COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar

RUN tar xf ${APPLICATION}.tar --strip-components=1 && rm -rf ${APPLICATION}.tar

FROM airbyte/integration-base-java:dev

ENV APPLICATION destination-dynamodb

COPY --from=build /airbyte /airbyte

LABEL io.airbyte.version=0.1.0
LABEL io.airbyte.name=airbyte/destination-dynamodb
```

If this solution OK, I will update all Dockerfile(s)